apps/examples/wc-react18/public/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I think we can remove this file I don't think we're going to use any of the benefits it provides like PWA at least for now. we need to remove its import in index.html if this makes sense.